task-sdk/src/airflow/sdk/__init__.py

```diff
@@ -112,6 +114,7 @@
     "task": ".definitions.decorators",
     "task_group": ".definitions.decorators",
     "teardown": ".definitions.decorators",
+    "macros": ".definitions.macros",
```

Comment:
I don't think we need this lazy import -- users should not be importing macro modules directly anyway, they should only use it via `{{ macros.foo }}` in templates, so we don't need the convince import here.

Reply:
It was based on this comment: https://github.com/apache/airflow/pull/50642#discussion_r2090879373. To an end user it would not really matter as they access it using templates anyways. 

Maybe its just morally correct to have it in `airflow.sdk.macros` instead?

Reply:
Yeah let me remove that, better handled this way